packages/chaire-lib-backend/src/services/routing/Routing.ts

```diff
@@ -121,60 +121,77 @@ export class Routing {
         const { isCancelled } = options;
 
         // ** backend
-        const routingResult: TransitOrRouteCalculatorResult[] = [];
+        const routingPromises: Promise<TransitOrRouteCalculatorResult>[] = [];
```

Comment:
Should be routingResultPromises

Reply:
Could be. Probably not worth it to change it. The function is small enough that it's easy to see what it is.